e2e/page.ts

```diff
@@ -48,6 +64,19 @@ export default class PlaywrightDevPage {
         await expect(this.page.locator('#calls-widget')).toBeVisible();
     }
 
+    async endCall() {
+        await this.sendSlashCommand('/call end');
+        let modal = this.page.locator('#calls-end-call-modal');
+        if (await modal.isVisible()) {
+            await modal.getByRole('button', {name: 'End call'}).click();
+        } else {
+            modal = this.page.locator('.modal-content');
+            if (await modal.isVisible()) {
+                await modal.getByRole('button', {name: 'Understood'}).click();
+            }
+        }
```

Comment:
Why is the check necessary? Generally I'd expect it to fail if we cannot end the call because our assumption of the state is likely not correct.

Reply:
Because we're running it in `test.afterEach` we have to handle either case. If we cannot end the call (some weird state or the server is broken) then that's fine. This is just the two happy path cases. 
The purpose is to clean up after tests that fail in the middle and then leave calls running, which screws up next tests and cause more failures. I want the only failure to be the test that failed for real reasons.

Reply:
That's fine. We can always create a separate method to make it stricter.

Reply:
I'm not sure what you mean -- what kind of separate method?